documentation/source/index.rst

```diff
@@ -1,37 +1,164 @@
-**********************************
+##################################
 Welcome to Cocotb's documentation!
-**********************************
+##################################
 
-Contents:
+..
+   This documentation tries to follow https://www.divio.com/blog/documentation/ (Daniele Procida)
+   Other media about the same topic:
+   - https://ep2018.europython.eu/media/conference/slides/get-your-documentation-right.pdf
+   - https://www.youtube.com/watch?v=t4vKPhjcMZg
+   - A good example: http://docs.django-cms.org/en/latest/contributing/documentation.html#contributing-documentation
+
+   See also https://github.com/cocotb/cocotb/wiki/Howto:-Writing-Documentation
 
 .. toctree::
    :maxdepth: 2
 
    introduction
```

Comment:
It's tempting to put an "installation" section here, rather than as a tutorial - that is, split the quickstart in two.

Not sure how much we want to squeeze into one PR though.

Reply:
Added a TODO for now.